js/client.js

```diff
@@ -85,6 +85,235 @@ export const ClientAPI = superclass =>
             const contents = await this.get(url, options);
             return contents;
         }
+
+        /**
+         * Returns a project of the provided client.
+         *
+         * @memberof ClientAPI
+         * @param {String} name The name of the client.
+         * @param {String} project The name of the project.
+         * @returns {Promise} The requested client's project.
+         */
+        async getClientProject(name, project) {

```

Comment:
I'm not sure about the naming of these functions? I believe it should be `getProjectClient()` Do we have the reference standard we're copying?